lib/src/react16_suggestors/react_style_maps_updater.dart

```diff
@@ -0,0 +1,393 @@
+              flagAsVariable();
+            }
+          } else if (cssPropertyValue is MethodInvocation) {
+            var invocation = cssPropertyValue;
+            // Handle `toRem(1).toString()`
+            if (invocation.methodName.name == 'toString' &&
+                invocation.target is MethodInvocation) {
+              invocation = invocation.target;
+            }
+
+            if (!const ['toPx', 'toRem'].contains(invocation.methodName.name)) {
+              flagAsVariable();
+            }
+          } else if (cssPropertyValue is DoubleLiteral ||
+              cssPropertyValue is IntegerLiteral) {
+            // do nothing
+          } else {
+            flagAsVariable();
+          }
+        });
+      } else if (stylesObject is SimpleIdentifier) {
+        isAVariable = true;
+      } else if (stylesObject is MethodInvocation) {
+        isAFunction = true;
+      } else {
+        isOther = true;
+      }
+
+      // Patch the comment at the top of the style map based upon the
+      // edits made.
+      if (listOfVariables.isNotEmpty || isAVariable || isAFunction || isOther) {
+        yieldPatch(
+            cascade.offset,
+            cascade.offset,
+            getString(
+                isAVariable: isAVariable,
+                styleMapContainsAVariable: styleMapContainsAVariable,
+                isAFunction: isAFunction,
+                affectedValues: listOfVariables,
+                addExtraLine: sourceFile.getLine(cascade.offset) ==
+                    sourceFile.getLine(cascade.parent.offset),
+                isOther: isOther));
+      }
+    }
+  }
+}
+
+String getString({
+  String styleMap,
+  List<String> affectedValues = const [],
+  bool isAVariable = false,
+  bool styleMapContainsAVariable = false,
+  bool isAFunction = false,
+  bool addExtraLine = false,
+  bool isForCustomProps = false,
+  bool isOther = false,
+}) {
+  String checkboxWithAffectedValues = '// [ ] Check this box upon manual '
+      'validation that this style map uses a valid value for the following '
+      'keys: ${affectedValues.join(', ')}.';
+
+  String variableCheckbox = '// [ ] Check this box upon manual validation '
+      'that this style map is receiving a value that is valid for the keys '
+      'that are simple string variables.';
+
+  String variableCheckboxWithAffectedValues = '// [ ] Check this box upon '
+      'manual validation that this style map is receiving a value that is '
+      'valid for the following keys: ${affectedValues.join(', ')}.';
+
+  String willBeRemovedSuffix = '//$willBeRemovedCommentSuffix';
+
+  String functionCheckbox = '// [ ] Check this box upon manual validation '
+      'that the method called to set the style prop does not return any '
+      'simple, unitless strings instead of nums.';
+
+  if (isAVariable || styleMapContainsAVariable || isOther) {
+    if (affectedValues.isNotEmpty) {
+      return '''
```

Comment:
When testing, I noticed that these comment could take up a lot of space and be quite distracting. We might want to consider shortening them.

Perhaps make `styleMapExample` single line?

Reply:
Example: https://github.com/Workiva/graph_app/compare/react-16-codemod-initial-test?w=1#diff-8e5313e8d17c90e73135e146eb46d5deR203

Reply:
idk thats a longgg string
=